controlplane/pkg/common/composite.go

```diff
@@ -24,23 +24,23 @@ import (
 
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
-	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/common"
 )
 
 // CompositeNetworkService is the base service composition struct
 type CompositeNetworkService struct {
-	services []networkservice.NetworkServiceServer
+	services    []networkservice.NetworkServiceServer
+	factoryName string
```

Comment:
As near as I can tell... this is just used for building the name of the span?  Is that correct?

Reply:
Yes, to show it in logs and in jaeger, it is a just a logical string to represent.